src/main/api/opds.ts

```diff
@@ -66,31 +66,27 @@ export class OpdsApi {
 
     public async browse(data: any): Promise<OpdsResultView> {
         const { url } = data;
-        const opdsFeedData = await httpGet(url);
-        // let opds2Publication: OPDSPublication = null;
+        const opdsFeedData = await httpGet(url, {
```

Comment:
Side question about `const { url } = data;` and `browse(data: any)` ... do you know what the TypeScript type is for `data`?

Reply:
I am merging this PR without a code change for `any`. This can be resolved at a later stage.